bin/zopen-build

```diff
@@ -464,6 +466,8 @@ extractTarBall()
   if ! git init . >/dev/null || ! git add -f ${files} >/dev/null || ! git commit --allow-empty -m "Create Repository for patch management" >/dev/null; then
     printError "Unable to initialize git repository for tarball"
   fi
+ 
```

Comment:
Minor: extra spaces?

Reply:
i am not seeing them... where do you see the extra spaces?

Reply:
I mean extra newlines (line 469 and 470)?